pkg/globalnet/controllers/uninstall.go

```diff
@@ -52,6 +52,7 @@ func UninstallDataPath() {
 		// The chains have to be deleted in a specific order.
 		constants.SmGlobalnetEgressChainForCluster,
 		constants.SmGlobalnetEgressChainForHeadlessSvcPods,
+		constants.SmGlobalnetEgressChainForHeadlessSvcEPs,
```

Comment:
Also, added uninstall code here. 

By this change,  this version of subctl comand starts to try deleting this chain, while old globalnet deployment doesn't have this chain. However,  uninstallation continues even `FlushIPTableChain` fails, so there won't be issue by version mismatch. 

Actually, [`ClearChain`](https://github.com/coreos/go-iptables/blob/master/iptables/iptables.go#L427) called from `FlushIPTableChain` doesn't seem to return error even when the chain doesn't exist already (it creates new one and flush). So, it would be safe to just add the chain.